GateMaterials.db

```diff
@@ -108,29 +179,71 @@ Plexiglass: d=1.19 g/cm3; n=3; state=solid
         +el: name=Carbon     ; f=0.599848
         +el: name=Oxygen     ; f=0.319614
 
-GSO: d=6.7 g/cm3; n=3 ; state=Solid
+GSO: d=6.7 g/cm3; n=3 ; state=solid
         +el: name=Gadolinium ; n=2
         +el: name=Silicon    ; n=1
         +el: name=Oxygen     ; n=5
 
-LuAP: d=8.34 g/cm3; n=3 ; state=Solid
+LuAP: d=8.34 g/cm3; n=3 ; state=solid
         +el: name=Lutetium   ; n=1
         +el: name=Aluminium  ; n=1
         +el: name=Oxygen     ; n=3
 
-YAP: d=5.55 g/cm3; n=3 ; state=Solid
+YAP: d=5.55 g/cm3; n=3 ; state=solid
         +el: name=Yttrium    ; n=1
         +el: name=Aluminium  ; n=1
         +el: name=Oxygen     ; n=3
 
+Lucite: d=1.19 g/cm3; n=3;
+        +el: name=Hydrogen   ; f=0.080538
+        +el: name=Carbon     ; f=0.599848
+        +el: name=Oxygen     ; f=0.319614
+
+CarbonFiber: d=1.75 g/cm3; n=2; state=Solid
+        +el: name=Carbon     ; f=0.977
+        +el: name=Oxygen     ; f=0.023
+
 Water: d=1.00 g/cm3; n=2 ; state=liquid
         +el: name=Hydrogen   ; n=2
         +el: name=Oxygen     ; n=1
 
-Quartz: d=2.2 g/cm3; n=2 ; state=Solid
+Quartz: d=2.2 g/cm3; n=2 ; state=solid
         +el: name=Silicon    ; n=1
         +el: name=Oxygen     ; n=2
 
+Fe3O4: d=5.17 g/cm3; n=2; state=Solid
+        +el: name=Iron       ; n=3
+        +el: name=Oxygen     ; n=4
+
+Mylar: d=1.4 g/cm3; n=3; state=solid
+        +el: name=Hydrogen   ; f=0.041958
+        +el: name=Carbon     ; f=0.625017
+        +el: name=Oxygen     ; f=0.333025
+
+Kapton: d=1.42 g/cm3; n=4; state=solid
+        +el: name=Hydrogen   ; f=0.026362
+        +el: name=Carbon     ; f=0.691133
+        +el: name=Nitrogen   ; f=0.073270
+        +el: name=Oxygen     ; f=0.209235
+
+Nomex: d=0.95 g/cm3; n=4; state=solid
+        +el: name=Hydrogen   ; n=10
+        +el: name=Carbon     ; n=14
+        +el: name=Nitrogen   ; n=2
+        +el: name=Oxygen     ; n=2
+
+NitrogenGas: d=1.29 mg/cm3 ; n=1 ; state=gas
+        +el: name=Nitrogen   ; f=1
+
+Hostapan: d=1.4 g/cm3; n=3; state=solid
```

Comment:
Any clue what that is?

Reply:
According to French wikipedia, Hostaphan (with ph instead of p) one of the brand names under which a foil of "Polytéréphtalate d'éthylène" (a.k.a. PET) is known. Another name for exactly the same material is Mylar. I think the name Mylar is much more common, so I am keeping Mylar and deleting "Hostapan".